test/HardwareObjectsMockup.xml/mxcube-web/server.yaml

```diff
@@ -17,6 +17,9 @@ mxcube:
   USE_EXTERNAL_STREAMER: True
   VIDEO_FORMAT: MPEG1
   VIDEO_STREAM_URL: "ws://localhost:8000/ws"
+  
+  # At which port to stream from
+  VIDEO_STREAM_PORT: 8000
```

Comment:
The `VIDEO_STREAM_URL` is used to tell the front end from which URL to get the video stream and does not necessarily include any port (in the case of proxy)

The `VIDEO_STREAM_PORT` is used internally for the video_streamer to tell which port to stream from, the streamer runs on localhost (in a separate process) 

The same configuration value `VIDEO_STREAM_URL` was previously used for both means but that only works if everything runs on localhost. So the two configuration keys/values were separated.